lib/mutant/cache.rb

```diff
@@ -21,7 +24,7 @@ def initialize
     # @api private
     def parse(path)
       @cache.fetch(path) do
-        @cache[path] = Parser::CurrentRuby.parse(File.read(path))
+        @cache[path] = Parser::CurrentRuby.parse(file.read(path))
```

Comment:
> So this file object is actually a ::File class? Why not have your path object be a Pathname? That's better than passing around String objects to represent a file path.

Its the `::File` singleton, I want to inject it for easier testing as setting up a message expectation on an IO global is a bad idea.

Pathname does not have `Pathname.read(a_file)` interface.